school/diary/views.py

```diff
@@ -1,88 +1,142 @@
+            <input name="fee" id="new_item" value="" />
+        </div>
+            <button>Send my choice</button>
     </form>
+"""
+
+update_template = f"""
+    <form action="/param_update/" method="POST">
+        <div>
+            <label for="new_item">Enter new name of the player</label>
+            <input name="name" id="new_item" value="" />
+        </div>
+        <p>
+        <div>
+            <label for="new_item">Please specify the position of the player (e.g. GK, CB, CMF, ST, LW, CAM)</label>
+            <input name="position" id="new_position" value="" />
+        </div>
+        <p>
+        <fieldset>
+            <legend>Select new football club:</legend>
+            <div>
+                <input type="radio" id="contactChoice1" name="contact" value="PSG"  />
+                <label for="contactChoice1">PSG</label>
+                <input type="radio" id="contactChoice2" name="contact" value="Parma" />
+                <label for="contactChoice2">Parma</label>
 
+                <input type="radio" id="contactChoice3" name="contact" value="Juventus" />
+                <label for="contactChoice2">Juventus</label>
+            </div>
+        </fieldset>
+        <p>
+        <div>
+            <label for="new_item">Specify new market value</label>
+            <input name="fee" id="new_item" value=""/>
+        </div>
+        <p>
+            <button>Send my choice</button>
+        <p>
+        <div>
+            <a href="/">Return to the HOME page</a>
+        </div>
+    </form>
 """
 
 
+user_pk = 4
+
+
 def index(request):
     return HttpResponse(index_template)
 
 
-# def login(request):
-#     if request.POST:
-#         name = request.POST.get("name")
-#         position = request.POST.get("position")
-#         club = request.POST.get("contact")
-#         transfer_fee = request.POST.get("transfer_fee")
-#         response = f"Player Name: {name}, Position: {position}, Club: {club}, " \
-#                    f"Transfer fee: {transfer_fee} millions euros"
-#     else:
-#         response = "Please create a player"
-#     return HttpResponse(response)
-
 def login(request):
     if request.POST:
         name = request.POST.get("name")
         position = request.POST.get("position")
         club = request.POST.get("contact")
         user_input = False
         try:
-            transfer_fee = request.POST.get('fee')
+            transfer_fee = int(request.POST.get('fee'))
             user_input = True
         except ValueError:
             transfer_fee = random.randrange(1, 100)
         response = f"Player Name: {name}, Position: {position}, Club: {club}, " \
                    f"Transfer fee {'(added by user): ' if user_input else '(random integer): '}{transfer_fee} millions euros"
+        print(response)
+        diary_football_player = FootballPlayer(name=name, position=position, club=club, transfer_fee=transfer_fee)
+        diary_football_player.save()
```

Comment:
From our conversation, you should store the state of the login user in the global scope for example:
```
global user_pk
user_pk = diary_football_player.id
```

so on the following steps when you will update the data you could do something like this:
```
if user_pk:
    diary_football_player = FootballPlayer.objects.get(pk=user_pk)
    ...
```